src/vulkan-renderer/wrapper/command_buffer.cpp

```diff
@@ -4,13 +4,15 @@
 #include "inexor/vulkan-renderer/wrapper/resource_descriptor.hpp"
 
 #include <array>
+#include <cassert>
 #include <stdexcept>
 #include <utility>
 
 namespace inexor::vulkan_renderer::wrapper {
 
-CommandBuffer::CommandBuffer(const wrapper::Device &device, VkCommandPool command_pool, const std::string &name)
+CommandBuffer::CommandBuffer(const wrapper::Device &device, const VkCommandPool command_pool, const std::string &name)
     : m_device(device), m_name(name) {
+    assert(device.device());
```

Comment:
This assert seems quite useless since the device wrapper can never be in a state where `device.device()` is `nullptr`.

Reply:
Right. Done.